images/Dockerfile.openshift

```
@@ -0,0 +1,23 @@
+# This dockerfile is specific to building Multus for OpenShift
+# The okd-builder image is locally built from https://raw.githubusercontent.com/okd-project/images/main/okd-builder.Dockerfile
+FROM local/okdbuilder:latest as builder
+
+ADD . /usr/src/multus-cni
```

Hadolint (2.15.1)

[error] Use COPY instead of ADD for files and folders

(DL3020)